src/main/java/edu/cornell/kfs/fp/document/CuDisbursementVoucherDocument.java

```diff
@@ -698,157 +735,177 @@ public boolean generateDocumentGeneralLedgerPendingEntries(GeneralLedgerPendingE
+             (!KFSConstants.COUNTRY_CODE_UNITED_STATES.equals(getWireTransfer().getBankCountryCode()) && wireCharge.getForeignChargeAmt().isZero()) ){
+            //DV is for a US bank and wire charge value is zero dollars OR DV is for a foreign bank and wire charge is zero dollars.
+            return true;
         }
+        return false;
+    }
 
-        @Override
-        public boolean answerSplitNodeQuestion(String nodeName) throws UnsupportedOperationException {
-            if (nodeName.equals(CuDisbursementVoucherDocument.DOCUMENT_REQUIRES_AWARD_REVIEW_SPLIT))
-                return isCAndGReviewRequired();
-            if (nodeName.equals(CuDisbursementVoucherDocument.DOCUMENT_REQUIRES_CAMPUS_REVIEW_SPLIT))
-                return isCampusReviewRequired();
-            if (nodeName.equals(DisbursementVoucherDocument.DOCUMENT_REQUIRES_TAX_REVIEW_SPLIT)) {
-                return isTaxReviewRequired();
-            }
-            if (nodeName.equals(DisbursementVoucherDocument.DOCUMENT_REQUIRES_TRAVEL_REVIEW_SPLIT)) {
-                return isTravelReviewRequired();
-            }
-            if (nodeName.equals(DOCUMENT_REQUIRES_SEPARATION_OF_DUTIES)) {
-                return isSeparationOfDutiesReviewRequired();
-            }
-            throw new UnsupportedOperationException("Cannot answer split question for this node you call \""+nodeName+"\"");
+    @Override
+    public boolean answerSplitNodeQuestion(String nodeName) throws UnsupportedOperationException {
+        if (nodeName.equals(CuDisbursementVoucherDocument.DOCUMENT_REQUIRES_AWARD_REVIEW_SPLIT))
+            return isCAndGReviewRequired();
+        if (nodeName.equals(CuDisbursementVoucherDocument.DOCUMENT_REQUIRES_CAMPUS_REVIEW_SPLIT))
+            return isCampusReviewRequired();
+        if (nodeName.equals(DisbursementVoucherDocument.DOCUMENT_REQUIRES_TAX_REVIEW_SPLIT)) {
+            return isTaxReviewRequired();
+        }
+        if (nodeName.equals(DisbursementVoucherDocument.DOCUMENT_REQUIRES_TRAVEL_REVIEW_SPLIT)) {
+            return isTravelReviewRequired();
         }
+        if (nodeName.equals(DOCUMENT_REQUIRES_SEPARATION_OF_DUTIES)) {
+            return isSeparationOfDutiesReviewRequired();
+        }
+        throw new UnsupportedOperationException("Cannot answer split question for this node you call \""+nodeName+"\"");
+    }
 
-        public boolean isTravelReviewRequired() {
-            List<AccountingLine> theList = (List<AccountingLine>) this.sourceAccountingLines;
+    public boolean isTravelReviewRequired() {
+        List<AccountingLine> theList = (List<AccountingLine>) this.sourceAccountingLines;
 
-            for (AccountingLine alb : theList )
+        for (AccountingLine alb : theList )
+        {
+            ParameterEvaluator objectCodes = SpringContext.getBean(ParameterEvaluatorService.class).getParameterEvaluator("KFS-FP", "DisbursementVoucher", OBJECT_CODES_REQUIRING_TRAVEL_REVIEW, alb.getFinancialObjectCode());
```

Comment:
I could. I was focusing on the area of code I was changing (the only change here was reformatting) which is why I hadn't done that here.